### PR TITLE
Localization improvements

### PR DESCRIPTION
**Pull request by tbnv**
- Wrapped missing strings
- Typos and labels standardization (no ":" like WP default, etc.)
- Added French localization

### CODE REVIEW

File: adminpages/addmember.php [hunk 1]
```diff
@@ -4,8 +4,8 @@
 	if(!function_exists("current_user_can") || (!current_user_can("manage_options") && !current_user_can($cap)))
 	{
 		die(__("You do not have permissions to perform this action.", "pmpro"));
```
Comment (andrewlimaza) [inline adminpages/addmember.php]: This should use 'pmpro-add-member-admin'

Comment (andrewlimaza) [inline adminpages/addmember.php]: This can use pmpro-add-member-admin text domain.

**Comment by tbnv** [comment]
Didn't spot this one, thanks. Fixed.

**Comment by tbnv** [comment]
Let me know if there is still any issue with this.
Can't wait to update the plugin on our WP install. 🙂

**Pull request merged**